1.0/docs/start/psk/setting-up.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Setting up
+subtitle: Polymer Starter Kit
+---
+
```

Comment:
Usually start every article with intro text before the first header. In this case, it would be a good idea to explain what the PSK is before we launch into installing it. i.e., Why do I want to set up the PSK?